examples/grpc/Earthfile

```diff
@@ -0,0 +1,39 @@
+FROM node:13.10.1-alpine3.11
+
+WORKDIR /example-grpc
+
+# This example shows how to use earthly to compile a protobuf grpc definition
+# into protobuf code for both a Go-based server, and a python-based client.
+#
+# This example spawns 3 different git repos:
```

Comment:
```suggestion
# This example spans 4 different git repos:
```

(The current file is in the 4th repo - might be clearer to the reader)

Reply:
lol, I must have been thinking about fish. I fixed this and slightly reworded the last 4th numbered item.